client/src/App/reducers/users.js

```diff
@@ -13,6 +13,12 @@ const users = (state = {
                 ...state,
                 isFetching: true
             };
+        case DELETE_USER:
+            const userId = action.user._id;
+            return Object.assign({},
+                state, { users: state.users.filter(user => user._id !== userId)}
```

Comment:
this looks good to me. One style note - I know that I use Object.assign places, but some people thing Object.assign is bad because it's not readable. 

Probably dont worry about this now, but something for us both to keep in mind going forward.